src/composables/popupProps.ts

```diff
@@ -22,7 +24,7 @@ export function usePopupProps(): IPopupPropsState {
 
   const sender = computed((): IAccountOverView => ({
     name: popupProps.value?.app?.name,
-    address: popupProps.value?.app?.host,
+    address: popupProps.value?.app?.host as Encoded.AccountAddress,
```

Comment:
App host is an AccountAddress? Looks like a mistake.

Reply:
`IAccountOverview` type is used. Which is based on `IAccount` which is used for the accounts, where address is supposed to be `ak_...`. The question is why it was used in the first place. Maybe it can be done separately.

Reply:
I suggest to change the type of `address` in the `IAccountOverview` to `Encoded.AccountAddress | string` for now.
And maybe add some comment to this prop in the types file that the `string` stands for URL.

In the future we should make a new interface for this `sender` stuff. What do you think?